Marsiple/App/Todo Form/TodoFormView.swift

```diff
@@ -38,16 +31,13 @@ class TodoFormView: UIView {
 
 extension TodoFormView: UITextViewDelegate {
     func textViewDidBeginEditing(_ textView: UITextView) {
-        if textView.textColor == .lightGray {
-            textView.text = nil
-            textView.textColor = .black
+        if textView.text == placeholder {
+            isPlaceholderActive(false)
```

Comment:
@dkulundzic That wouldn't work when `placeholder` is active, because then it would just color the `placeholder` text black. I could add `if newValue == placeholder` to the method, but I don't know if that is a better solution.

Reply:
How about `isPlaceholderActive(false, newValue: textView.text == placeholder ? nil: textView.text)`?

Reply:
Ok, that would work.